doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -316,15 +336,15 @@ identᵢ’.
     Check tree_size_ind2.
 
 .. _derive-inversion:
-     
+
 Generation of inversion principles with ``Derive`` ``Inversion``
 -----------------------------------------------------------------
 
 The syntax of ``Derive`` ``Inversion`` follows the schema:
 
 .. cmd:: Derive Inversion @ident with forall (x : T), I t Sort sort
 
-This command generates an inversion principle for the `inversion … using` 
+This command generates an inversion principle for the `inversion … using`
```

Comment:
This description is not correctly indented. It would be best to reformulate the paragraph so that the sentence ```The syntax of ``Derive`` ``Inversion`` follows the schema:``` disappears.